tutorials/Working_with_fNIRS_data.py

```diff
@@ -0,0 +1,123 @@
+"""
+.. _tut_fnirs_data
+
+=======================
+Use MNE with fNIRS data
+=======================
+
+MNE Python was implemented with EEG and MEG in mind, but can be useful to
```

Comment:
*originally* implemented

(nowadays we try to implement things with a broader scope)